graphql/database/users/__tests__/getShouldShowSfacIcon.test.js

```diff
@@ -13,60 +16,101 @@ beforeEach(() => {
+  })
+
+  it('returns false if the user is in the control group for the "new user SFAC notification" experiment', async () => {
+    expect.assertions(1)
+
+    const getUserFeature = require('../../experiments/getUserFeature').default
+    getUserFeature.mockImplementation((_userContext, _user, featureName) => {
+      if (featureName === SFAC_EXTENSION_PROMPT) {
+        return new Feature({
+          featureName: SFAC_EXTENSION_PROMPT,
+          variation: 'Control',
+        })
+      }
+      return new Feature()
+    })
 
     const getShouldShowSfacIcon = require('../getShouldShowSfacIcon').default
     const result = await getShouldShowSfacIcon(userContext, user)
     expect(result).toEqual(false)
   })
 
-  it('returns false if the user is in not the experiment but it is dev', async () => {
-    jest.doMock('../../../config', () => ({
-      GROWTHBOOK_ENV: 'dev',
-      DB_TABLE_NAME_APPENDIX: '',
-    }))
+  it('returns true if the user is in the treatment group for the "existing user activity icon" experiment', async () => {
+    expect.assertions(1)
+
+    const getUserFeature = require('../../experiments/getUserFeature').default
+    getUserFeature.mockImplementation((_userContext, _user, featureName) => {
+      if (featureName === SFAC_EXISTING_USER_ACTIVITY_ICON) {
+        return new Feature({
+          featureName: SFAC_EXISTING_USER_ACTIVITY_ICON,
+          variation: 'Icon',
+        })
+      }
+      return new Feature()
+    })
+
+    const getShouldShowSfacIcon = require('../getShouldShowSfacIcon').default
+    const result = await getShouldShowSfacIcon(userContext, user)
+    expect(result).toEqual(true)
+  })
+
+  it('returns false if the user is in the control group for the "existing user activity icon" experiment', async () => {
     expect.assertions(1)
 
     const getUserFeature = require('../../experiments/getUserFeature').default
-    getUserFeature.mockResolvedValueOnce(
-      new Feature({
-        featureName: SFAC_EXTENSION_PROMPT,
-        variation: 'Control',
-      })
-    )
+    getUserFeature.mockImplementation((_userContext, _user, featureName) => {
+      if (featureName === SFAC_EXISTING_USER_ACTIVITY_ICON) {
+        return new Feature({
+          featureName: SFAC_EXISTING_USER_ACTIVITY_ICON,
+          variation: 'Control',
+        })
+      }
+      return new Feature()
+    })
 
     const getShouldShowSfacIcon = require('../getShouldShowSfacIcon').default
     const result = await getShouldShowSfacIcon(userContext, user)
     expect(result).toEqual(false)
   })
 
-  it('returns true if the user is in the experiment and env is dev', async () => {
-    jest.doMock('../../../config', () => ({
-      GROWTHBOOK_ENV: 'dev',
-      DB_TABLE_NAME_APPENDIX: '',
-    }))
+  it('returns true if the user is in the treatment group for the "new user SFAC notification" experiment but in the control group for the "existing user activity icon" experiment', async () => {
```

Comment:
nit: do we want a test for new user SFAC notification = `control` and existing user activity group icon = `icon`?

Reply:
Sounds good, I'll add this and merge.